src/blockdata/transaction.rs

```diff
@@ -169,7 +154,8 @@ impl ::core::str::FromStr for OutPoint {
     type Err = ParseOutPointError;
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
-        if s.len() > 75 { // 64 + 1 + 10
+        if s.len() > 75 {
+            // 64 + 1 + 10
```

Comment:
I'm surprised that rustfmt moves comments :/. It used to be that you could suffix lines with `//` to force it not to mangle them.

In any case, the original comment placement was correct and the new placement is incorrect.

Reply:
Thanks for flagging this. I messed with it a bunch and couldn't get a satisfactory result and yes rustfmt is clearly wrong in this. Putting the comment above line 157 is all we can do, I did mean to manually move the comment, my bad.

Reply:
I would advise to use `/* smth */` next to the value (before or after, depending on the context), instead of `//` at the end of the line. It generally reads better, and formats better too.

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=f7e186d6f3d3c839a9ff59b9ff7458d1 and click "Tools -> Rustfmt"

Reply:
Oh it works if the comment is _before_ the 75 but not after the 75, that is a bit silly but far better than on the newline. Thanks.